Sources/Hummingbird/Router/Router.swift

```diff
@@ -128,3 +128,25 @@ public struct RouterOptions: OptionSet, Sendable {
     /// For every GET request that does not have a HEAD request, auto generate the HEAD request
     public static var autoGenerateHeadEndpoints: Self { .init(rawValue: 1 << 1) }
 }
+
+extension Router {
+    /// Route description
+    public struct RouteDescription: CustomStringConvertible {
+        /// Route path
+        public let path: RouterPath
+        /// Route method
+        public let method: HTTPRequest.Method
+
+        public var description: String { "\(method): \(path)" }
```

Comment:
Can we print it without colon? E.G.

```
GET /users/
POST /users/
```

```suggestion
        public var description: String { "\(method) \(path)" }
```